iothub/device/src/Authentication/HsmAuthentication/HttpHsmComunicationException.cs

```diff
@@ -4,7 +4,7 @@
 using System;
 using System.Runtime.Serialization;
 
-namespace Microsoft.Azure.Devices.Client.HsmAuthentication
+namespace Microsoft.Azure.Devices.Client
```

Comment:
Except for specific cases, normalizing all namespaces to the project's (not conforming to folders).